src/main.ts

```diff
@@ -3,7 +3,10 @@ import {
   requireEvent,
   requireFiles,
   requirePr,
-  requirePullNumber
+  requirePullNumber,
+  require_cc0,
+  require_created_at,
+  require_max_file_number
 } from "#/assertions";
 import { PullRequestUseCases } from "#/pull_request/use_cases";
```

Comment:
Does anyone know what the `#` represents in an import statement?  Maybe @alita-moore?

Reply:
It is the directory, defined as "#/*": ["src/modules/*"] 
`[EIP-Bot](https://github.com/ethereum/EIP-Bot)/tsconfig.json'
line 28

Reply:
Hmm, I'm surprised that works, but OK.